components/gitpod-cli/go.mod

```diff
@@ -12,7 +12,7 @@ require (
 	github.com/google/shlex v0.0.0-20181106134648-c34317bd91bf
 	github.com/google/tcpproxy v0.0.0-20180808230851-dfa16c61dad2
 	github.com/gorilla/handlers v1.5.1
-	github.com/manifoldco/promptui v0.3.2
+	github.com/manifoldco/promptui v0.9.0
```

Comment:
The newer version allows hiding the message after selection using `HideSelected`

Reply:
I also ran `go mod tidy` that's why these changes